From f8cd93ebb28a9b131f5f772d50b496bd7b1a2e2f Mon Sep 17 00:00:00 2001
From: Karthikeyan K <karthikk@habana.ai>
Date: Fri, 10 Jun 2022 04:47:15 +0300
Subject: [PATCH] [SW-89971] gaudi3: rom: deprecate erom tpm symbol

As we need to move tpm to the regular image descriptor(a.k.a spi image
descriptor) the index is deprecated for gaudi3 platform.

Change-Id: I0944c0f85aaa0149eb93ae2bc884e57a4723460d
---

diff --git a/common/hl_boot_desc_rom.h b/common/hl_boot_desc_rom.h
index 155e9f4..6be9424 100644
--- a/common/hl_boot_desc_rom.h
+++ b/common/hl_boot_desc_rom.h
@@ -20,7 +20,7 @@
 	HL_FW_EROM_IMG_ROOT = 0,
 	HL_FW_EROM_PCIE = 1,
 	HL_FW_EROM_RECOVERY = 2,
-	HL_FW_EROM_TPM = 3,
+	HL_FW_EROM_TPM = 3, /* Symbol deprecated for gaudi3 */
 	HL_FW_MAX_EROM_BOOT_PTRS,
 };
 
